nc_py_api/_exceptions.py

```diff
@@ -60,6 +60,8 @@ def check_error(response: Response, info: str = ""):
         else:
             phrase = "Unknown error"
         raise NextcloudException(status_code, reason=phrase, info=info)
-    if not codes.is_error(status_code):
```

Comment:
there is no strict equivalent at niquests `httpx.codes`
but the patch will act exactly as before.